packages/protobuf-test/src/codegenv1/boot.test.ts

```diff
@@ -104,6 +105,9 @@ describe("bootFileDescriptorProto()", () => {
     clearField(DescriptorProtoDesc, d, "reservedName");
     for (const f of d.field) {
       clearField(FieldDescriptorProtoDesc, f, "jsonName");
+      if (f.options) {
+        clearField(FieldOptionsDesc, f.options, "featureSupport");
+      }
```

Comment:
We embed the file descriptor for google/protobuf/descriptor.proto, and cannot use binary serialization during bootstrap. Since the new field google.protobuf.FieldOptions.feature_support is not relevant for serialization, we drop it.